docs/manual/nullness-checker.tex

```diff
@@ -122,6 +122,12 @@
   checker will unsoundly assume that the argument to \<Map.get> is a key
   for the receiver map.  It will not do any checking of
   \refqualclass{checker/nullness/qual}{KeyFor} and related qualifiers.
+
+\item
+  \label{nullness-conservativeArgumentNullnessAfterInvocation}%
+  If you supply the \code{-AconservativeArgumentNullnessAfterInvocation} command-line option, then the
+  checker will assume receiver and params are nullable after the method invocation. When running gradle
+  tests we disable this option by default except when doing \code{checkNullness} test.
```

Comment:
Is this important to readers of the manual?

Reply:
No, as the task `checkNullness` is for our own testing purpose. I will delete this line.